typing/env.ml

```diff
@@ -1242,7 +1261,7 @@ let make_copy_of_types env0 =
     IdTbl.map f env0.values
   in
   (fun env ->
-     if env.values != env0.values then fatal_error "Env.make_copy_of_types";
+     (*if env.values != env0.values then fatal_error "Env.make_copy_of_types";*)
```

Comment:
This assertion fails with `remove_last_open`. Can we replace it with something else?